sdk/iot/hub/samples/src/generate_certificate.sh

```diff
@@ -22,5 +22,8 @@ echo -e "\nSample certificate generated"
 echo "Use the device_cert_store.pem file within the sample:"
 echo    export AZ_IOT_DEVICE_X509_CERT_PEM_FILE=$(pwd)/device_cert_store.pem
 
-echo -e "\nUse the following fingerprint when creating your device in IoT Hub (must remove colons):"
-openssl x509 -noout -fingerprint -in device_ec_cert.pem
+echo -e "\nUse the following fingerprint when creating your device in IoT Hub:"
+FINGERPRINT=`openssl x509 -noout -fingerprint -in device_ec_cert.pem`
```

Comment:
>openssl x509 -noout -fingerprint -in device_ec_cert.pem [](start = 13, length = 55)

openssl x509 -noout -fingerprint -in device_ec_cert.pem | tee | sed 's/://g' > fingerprint.txt

Reply:
In windows you can open these cert by double click and get the hash. (by just renaming pem -> crt)

---
In reply to: [421902872](https://github.com/Azure/azure-sdk-for-c/pull/702#discussion_r421902872) [](ancestors = 421902872)

Reply:
To comment one: Is that just another way of doing it? Does it add a benefit?
To two: Yes this just functions as a catch all solution

Reply:
yes, simpler and no need to create variables.

Reply:
Gotcha good to know thanks. Updated